src/types.tsx

```diff
@@ -1,27 +1,30 @@
 import React from 'react';
 
 export interface QuestionFormat{
```

Comment:
I'm wondering if the issues with typing that we're getting have to do with this interface structure and maybe breaking it down so that it isn't nested in the definition. Maybe make attributes it's own interface that we assign to QuestionFormat.

Reply:
That is a good call, we can try to split it. Let's give it a try later today.